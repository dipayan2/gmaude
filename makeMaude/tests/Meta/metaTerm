#!/bin/sh

MAUDE_LIB=$srcdir/../../src/Main
export MAUDE_LIB

../../src/Main/maude \
  < $srcdir/metaTerm.maude -no-banner -no-advise \
  > metaTerm.out 2>&1

diff $srcdir/metaTerm.expected metaTerm.out > /dev/null 2>&1
