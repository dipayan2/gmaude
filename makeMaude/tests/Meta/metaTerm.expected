==========================================
reduce in QID : 'a:s .
rewrites: 0
result Qid: 'a:s
==========================================
reduce in META-TERM : 'a:s .
rewrites: 0
result Variable: 'a:s
==========================================
reduce in META-TERM : ''a:b .
rewrites: 0
result Variable: ''a:b
==========================================
reduce in META-TERM : getName('a.s) .
rewrites: 6
result Sort: 'a
==========================================
reduce in META-TERM : getType('a.s) .
rewrites: 7
result Sort: 's
==========================================
reduce in META-TERM : getName('a:s) .
rewrites: 6
result Sort: 'a
==========================================
reduce in META-TERM : getType('a:s) .
rewrites: 7
result Sort: 's
==========================================
reduce in META-TERM : '`[s`] .
rewrites: 0
result Kind: '`[s`]
==========================================
reduce in META-TERM : '`[s`,t`] .
rewrites: 0
result Kind: '`[s`,t`]
==========================================
reduce in META-TERM : getName('a.`[s`,t`]) .
rewrites: 6
result Sort: 'a
==========================================
reduce in META-TERM : getType('a.`[s`,t`]) .
rewrites: 7
result Kind: '`[s`,t`]
==========================================
reduce in META-TERM : getName('a:`[s`,t`]) .
rewrites: 6
result Sort: 'a
==========================================
reduce in META-TERM : getType('a:`[s`,t`]) .
rewrites: 7
result Kind: '`[s`,t`]
Bye.
